master/pkg/model/hyperparameters_config.go

```diff
@@ -14,6 +15,39 @@ const GlobalBatchSize = "global_batch_size"
 // Hyperparameters holds a mapping from hyperparameter name to its configuration.
 type Hyperparameters map[string]Hyperparameter
 
+// Validate implements the check.Validatable interface.
+func (h Hyperparameters) Validate() []error {
+	b, ok := h[GlobalBatchSize]
+	if !ok {
+		return []error{
+			errors.New("global_batch_size hyperparameter must be specified"),
+		}
+	}
+	switch {
+	case b.ConstHyperparameter != nil:
+		if !isNumeric(b.ConstHyperparameter.Val) {
+			return []error{
+				errors.New("global_batch_size must be a numeric value"),
```

Comment:
non-blocking: probably change this to `global_batch_size hyperparameter ...`

Reply:
ah yeah, removed it at some point because of line length.. I wish we would turn up line length to 120 in master and agent ha.